doc/platform-support.appendix.md

```diff
@@ -0,0 +1,11 @@
+# Platform Support {#appendix-platform-support}
+
+Currently, the `x86_64-linux` platform receives the highest level of support, both in terms of CI and of developer attention.
+The `aarch64-linux` plaform is also well supported, with most packages building succesfully in CI.
+
+On Darwin, `aarch64-darwin` receives better support than `x86_64-darwin`.
```

Comment:
```suggestion
- `x86_64-linux`: Highest level of support, both in terms of CI and of developer attention.
- `aarch64-linux`: Well supported, with most packages building successfully in CI.
- `aarch64-darwin`: Receives better support over `x86_64-darwin`
- `x86_64-darwin`: Receives some support
```

Reply:
Do you find this better? I personally prefer the more discursive style, in this case.

Reply:
Sometimes you do just want to scan through.

Could have the more verbose description and a more concise table showing support?

Reply:
Also the end state is probably a full table (eventually), so we can start in this direction anyway?

Reply:
Sounds good, I'll switch to a table.